dm/dm/worker/server.go

```diff
@@ -679,12 +679,15 @@ func (s *Server) enableHandleSubtasks(sourceCfg *config.SourceConfig, needLock b
 	}
 
 	if sourceCfg.EnableRelay {
-		w.startedRelayBySourceCfg = true
-		if err2 := w.EnableRelay(); err2 != nil {
+		log.L().Info("will start relay by `enable-relay` in source config")
+		if err2 := w.EnableRelay(true); err2 != nil {
 			log.L().Error("found a `enable-relay: true` source, but failed to enable relay for DM worker",
 				zap.Error(err2))
 			return err2
 		}
+	} else if w.startedRelayBySourceCfg {
+		log.L().Info("will disable relay by `enable-relay` in source config")
```

Comment:
```suggestion
		log.L().Info("will disable relay by `enable-relay:false` in source config")
```

this log is too confusing 🤔 (user may feel strange: why disable relay when i enable-relay)